python_cross_site_scripting/bingothon_tracker/auth.py

```
@@ -0,0 +1,138 @@
+        default_registration_template())
+    return send_auth_token_mail(domain, user, reverse('tracker:confirm_registration'), template, sender, token_generator, extra_context)
+
+
+def send_auth_token_mail(domain, user, viewURI, template, sender=None, token_generator=default_token_generator, extra_context=None):
+    if not sender:
+        sender = viewutil.get_default_email_from_user()
+    reset_url = make_auth_token_url(domain, user, viewURI, token_generator)
+    formatContext = {
+        'user': user,
+        'domain': domain,
+        'reset_url': mark_safe( reset_url ),
```

Ruff (0.13.3)

Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)